<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabularWidget</class>
 <widget class="QWidget" name="TabularWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datos tabulares</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#TabularWidget {
    border-image: url(:/img/fondo.png) 0 0 0 0 stretch stretch;
    background-color: #03312E;
}

QLabel {
    color: #A8F9FF;
    font-size: 10pt;
}

QPushButton {
    background-color: #037171;
    color: #121619;
    border-radius: 8px;
    padding: 6px 16px;
    font-weight: bold;
    min-width: 120px;
}

QPushButton:hover {
    background-color: #9AE5E6;
}

QPushButton:pressed {
    background-color: #A8F9FF;
}

QListWidget {
    background-color: rgba(3, 49, 46, 230);
    border: 1px solid #037171;
    border-radius: 6px;
    color: #A8F9FF;
}

QTableView {
    background-color: rgba(3, 49, 46, 230);
    border: 1px solid #037171;
    border-radius: 6px;
    color: #A8F9FF;
    gridline-color: #037171;
    selection-background-color: #037171;
    selection-color: #121619;
}

QHeaderView::section {
    background-color: #037171;
    color: #121619;
    padding: 4px;
    border: 1px solid #021616;
}

QGraphicsView {
    background-color: rgba(3, 49, 46, 230);
    border: 1px solid #037171;
    border-radius: 6px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="load_csv_button">
       <property name="text">
        <string>Cargar datos</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_top">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="columns_label">
       <property name="text">
        <string>Columnas:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="column_list">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="plot_columns_button">
       <property name="text">
        <string>Graficar columnas</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mainLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QTableView" name="data_table">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="plotsLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="plot_label_1">
         <property name="text">
          <string>Gráfico 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="plot_view1"/>
       </item>
       <item>
        <widget class="QLabel" name="plot_label_2">
         <property name="text">
          <string>Gráfico 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="plot_view2"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
